README.md

```diff
@@ -73,13 +73,13 @@ replacing `docker run swarm` with just `swarm`.
 
 ## Participating
 
-You can contribute to Docker Swarm in several different ways:
+You can [contribute](https://github.com/docker/swarm/blob/master/CONTRIBUTING.md) to Docker Swarm in several different ways:
```

Comment:
Actually, let's keep it like this, we want people to always visit the most up-to-date version (in case they visit from an older tag)